Sources/Lumina/Camera/Extensions/FocusHandlerExtension.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FocusHandlerExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera {
+  func handleFocus(at focusPoint: CGPoint) {
+    self.sessionQueue.async {
+      guard let input = self.videoInput else {
+        return
+      }
+      do {
+        if input.device.isFocusModeSupported(.autoFocus) && input.device.isFocusPointOfInterestSupported {
+          try input.device.lockForConfiguration()
+          input.device.focusMode = .autoFocus
+          input.device.focusPointOfInterest = CGPoint(x: focusPoint.x, y: focusPoint.y)
+          if input.device.isExposureModeSupported(.autoExpose) && input.device.isExposurePointOfInterestSupported {
+            input.device.exposureMode = .autoExpose
+            input.device.exposurePointOfInterest = CGPoint(x: focusPoint.x, y: focusPoint.y)
+          }
+          input.device.unlockForConfiguration()
+        } else {
+          self.delegate?.finishedFocus(camera: self)
+        }
+      } catch {
+        self.delegate?.finishedFocus(camera: self)
+      }
+    }
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/FocusHandlerExtension.swift#L36"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>FocusHandlerExtension.swift:36</code></td>
    </tr>
  </tbody>
</table>